### ISSUE TITLE
2.0.8 crashing 

### ISSUE THREAD

**Issue by CaptainFrosty** [chiehmin/MinMinGuard#187]
Force closes immediately on launch. 2.0.7 works. Galaxy S8 Plus SM-G55U on Nougat

**Comment by MSkjel** [comment]
Try 2.0.9. If not send me a logcat

**Comment by tomporter518** [comment]
I just installed 2.0.9 and still get the same exception as 2.0.8 upon starting the UI. I'm attaching what I believe to be the relevant logcat section.
[minminguard_2_0_9_logcat.txt](https://github.com/chiehmin/MinMinGuard/files/3254304/minminguard_2_0_9_logcat.txt)

**Comment by MSkjel** [comment]
Ok, patch comming up shortly

**Comment by tomporter518** [comment]
Hmm...the new 2.0.9 apk didn't change anything. Still get a similar (same?) exception/crash on opening the UI.
[minminguard_2_0_9_logcat_2.txt](https://github.com/chiehmin/MinMinGuard/files/3254525/minminguard_2_0_9_logcat_2.txt)

**Comment by MSkjel** [comment]
Do you have MinMinGuard enabled in the Xposed Installer?

**Comment by MSkjel** [comment]
Well, I guess I'll have to do that differently. Pushing a new patch now. Weird how android can be so different between devices and software

**Comment by tomporter518** [comment]
Yes, I do. And I have toggled, restarted. The behavior is the same.

**Comment by MSkjel** [comment]
Ok, try the latest

**Comment by tomporter518** [comment]
I have updated to 2.0.9-v70 on both of my devices. Everything starts properly for me now.
